README.md

```diff
@@ -1,31 +1,36 @@
-# Open Container Distributable Image Specification
+# Open Container Initiative Image Format Specification
 
-The [Open Container Initiative](http://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
+The OCI Image Format project creates and maintains the software shipping container image format spec (OCI Image Format). The goal of this specification is to enable the creation of interoperable tools for building, transporting, and preparing a container image to run.
 
-This OCI project is tasked with creating a software shipping container image format spec (OCI Image Format) with security and naming as components.
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a [filesystem serialization](serialization.md), and an [image configuration](serialization.md#image-json-description). At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more filesystem serialization archives that will be unpacked to make up the final runnable filesystem. The image configuration includes information such as application arguments, environments, etc. The combination of the image manifest, image configuration, and one or more filesystem serializations is called the "OCI Image".
 
-## Initial Proposal
+![](img/build-diagram.png)
```

Comment:
have you tested `make docs` with this?
I'm thinking this may need to be inline base64, just like the media-types.png